### PR TITLE
staterecover: adds blobscan to local stack

### PR DESCRIPTION
**Pull request by jpnovais**
This PR implements issue(s) #

At the moment there is an issue with Blobscan fetching blobs from Teku

Reproduction steps
```
./gradlew localStackForStateRecoverComposeUp
./gradlew coordinator:ethereum:blob-submitter:integrationTest --tests "*submission works with contract V6"
```

```
2024-11-04 09:41:51     0: failed to index head block
2024-11-04 09:41:51     1: Failed to parallel process slots from 244 to 264:
2024-11-04 09:41:51        - Error processing slots range 244-264. Slot 244 failed: Deserialization Error: missing field `v` at line 1 column 7402. Response: {"number":"0xe9","hash":"0x59754bb4295e0a339f4a56f5fd206d0a4144c68f6ca0dc31e2e97cc75ff4bb00","mixHash":"0x964b7b1b32a14b392db90e37ff353f85fe540269f10272d9ee2dff44c2c64e7d","parentHash":"0x48c2daf40aaa4c36e17d81d86bd478baa8ab347791780b43495b9982b5d5e2ed","nonce":"0x0000000000000000","sha3Uncles":"0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347","logsBloom":"0x00000000000000000000000000000000000000000000020000
```

Tried to update Teku, but stack takes longer to start due to missed attestations.

I would check with Teku team if the may know the reason.
### Checklist

* [ ] I wrote new tests for my new core changes.
* [ ] I have successfully ran tests, style checker and build against my new changes locally.
* [ ] I have informed the team of any breaking changes if there are any.

### CODE REVIEW

File: docker/compose.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ networks:
     ipam:
       config:
         - subnet: 11.11.11.0/24
-  l1-network:
+  l1network:
```
Comment (Filter94) [inline docker/compose.yml:13]: Why this rename? Wasn't `l1-network` more readable?

**Review by Filter94 — APPROVED** [review]

File: docker/compose.yml (hunk 1 above)
Comment (jpnovais) [reply docker/compose.yml:13]: Yes it is more readable,  But the `-` sometimes was causing issues on the network finding/creation at start time.

Comment (Filter94) [reply docker/compose.yml:13]: Is it a docker-compose bug?

Comment (jpnovais) [reply docker/compose.yml:13]: I suspect so, but I didn't have time to dig in properly

**Pull request merged**